docs/src/SUMMARY.md

```diff
@@ -0,0 +1,108 @@
+# Summary
+
+[Introduction](index.md)
+* [Installation](installation.md)
+* [Contributing](contributing.md)
+
+# Reference
+* [Multi-call binary](multicall.md)
```

Comment:
this list could be generated too, no ? :)
sorry to be a pain but i hate long static list :)